server/src/main/java/com/coincoinche/engine/GameStateBidding.java

```diff
@@ -62,11 +62,12 @@ public static GameStateBidding initialGameStateBidding(Player firstPlayer) {
 
   /**
    * Return a boolean indicating if the game state must change, i.e. if the game gets to a state
-   * other than the bidding state. This method makes sense if it is called after a move was applied
-   * to the game.
+   * other than the bidding state. This method makes sense if it's called after players have
+   * rotated. to the game.
    */
   @Override
   public boolean mustChange() {
+    // TODO nockty: handle the case when all players pass
```

Comment:
Indeed good catch, I think the code in this method is fine though no?
We need to handle this in `if (state.mustChange())` of the method moveWasApplied of CoincheGameRound no?

Reply:
True, `mustChange` should likely be only when we need to go to a trick phase. I'll see if I can do this right now.